exercism/elm-test-runner
# Update `elm-explorations/test`

Closes #42 

This PR updates what is necessary to update to v2 of `elm-explorations/test`:
- elm-test-rs version in the docker builder
- elm.json files in test data (not strictly necessary since the file is overwritten, but convenient)
- update of a failure message mentioning previous version of elm-test-rs
- update dependencies of `extract-test-code` (not strictly necessary either, but it was easy)

I also added a CI check that is using the script in `exercism/elm` to run the tests on all exercises.

I expect that script to fail right now, because the reference `elm.json` in the main branch is using `elm-explorations/test` v1, which is incompatible with the new `elm-test-rs`.
What I will do next (in a little while) is hardcode Dockerfile and the CI to run on the PR branch in https://github.com/exercism/elm/pull/551. That should pass the CI, and give us confidence that this can be merged. And then I will undo the hardcoding. 
Once the other PR is merged, I will re-trigger the CI here. 

Hopefully that all works out.
